parsers/IN_UP.py

```diff
@@ -24,21 +26,21 @@ def fetch_data(country_code='IN-UP', session=None):
 
     key_map = {
         'total hydro generation': 'hydro',
-        'total thermal up generation': 'thermal',
+        'total thermal up generation': 'geothermal',
```

Comment:
These are actual geothermal plants? Thermal normally refers to conventional fossil fuel generation, if we don't know the type breakdown (coal, gas etc) lets just set it to unknown.

Reply:
They can't be geothermal plants. If there are 2 unknown keys then they will overwrite each in the dictionary so that needs to be avoided.

Reply:
okay got it. setting it to unknown.